plugin/table/table.go

```diff
@@ -103,58 +103,96 @@ func (t *Plugin) Ping() osquery.ExtensionStatus {
 
 func (t *Plugin) Shutdown() {}
 
+// ColumnType is a strongly typed representation of the data type string for a
+// column definition. The named constants should be used.
+type ColumnType string
+
+// The following column types are defined in osquery tables.h.
+const (
```

Comment:
Should we add the rest? I'm not sure I've seen them in practice yet, but it probably can't hurt to have the other types. 
https://github.com/osquery/osquery/blob/e6fe15eb49660725e65dba1549932ed96e0a8c6e/osquery/core/sql/column.h#L86

Reply:
Will do